src/components/Plate/Plate.tsx

```diff
@@ -102,125 +136,131 @@ export const Plate = ({
   };
 
   return (
-    <SelectableGroup
-      onEndSelection={handleSelection}
-      tolerance={selectionTolerance}
-      className={classNames("", className)}
-      selectingClassName="transition-all duration-100 ease-in-out animate-pulse "
-    >
-      <div
-        className={classNames(
-          "grid gap-2  ",
-          "text-xs md:text-sm lg:text-base",
-          wells > 96 && "px-4",
-          `grid-cols-${numCols} gap-2`,
-        )}
-        style={{
-          gridTemplateColumns: `repeat(${numCols}, minmax(0, 1fr))`,
-        }}
+    <>
+      <AnnotationBar
```

Comment:
Just a question: We are passing around a lot of state amongst all of the sub-components and state + UI is really mixed. Is it time to introduce a usePlateContext sort of thing where we shove all this state and then can access it directly in each of the sub-components? This would really cut down on the amount of props being passed around and might make things more readable.

Reply:
I think this is a good idea in the long run, but don't feel it's quite the right time. Let's keep this in mind as we go forward!